db/DB.js

```diff
@@ -171,7 +172,7 @@ const DB = {
         // Always add a verification_token value
         const verification_token = uuidv4();
         const res = await knex("subscribers")
-          .insert({ primary_sha1: sha1, primary_email: email, signup_language, primary_verification_token: verification_token, primary_verified: verified })
+          .insert({ primary_sha1: getSha1(email.toLowerCase()), primary_email: email, signup_language, primary_verification_token: verification_token, primary_verified: verified })
```

Comment:
I'm confused here. What's the difference between `sha1` (old) vs `getSha1(email.toLowerCase())` (new)?
I was thinking that `sha1` argument was now unused, but it looks like we still use it above on L155;

```js
await this._getSha1EntryAndDo(sha1, async aEntry => {...})
```

The only place I could find that calls this `async _addEmailHash() {}` method is below on L198;

```js
const emailHash = await this._addEmailHash(getSha1(email), email, signupLanguage, true);
```

Which feels confusing, because we are now passing `getSha1(email)` and then internally doing `getSha1(email.toLowerCase())`.

Reply:
The objective is to store the sha1 of the *lowercase* version of the email address.

This code is kinda complex and I'll see if I can clean this up while I'm in here. But if that turns into a rabbit-hole, I'd at least like to keep this `getSha1(email.toLowerCase())` fix to the existing bug.